src/components/chef-detail/chef-detail.tsx

```diff
@@ -17,16 +25,28 @@ export async function ChefDetail({
   recipeCount: number;
 }) {
   const session = await getServerSession(authOptions);
+  if (!session || !session.user) {
+    notFound();
+  }
+  const userLinks = chefInfo.userLinks.map((userLink, index) => ({ ...userLink, index }));
+
   return (
     <div className="flex w-full flex-col px-4 pt-4">
-      <div className="mb-2 flex items-start justify-between">
+      <div
+        className={cn(
+          "mb-2 flex items-start",
+          session && session.user && chefInfo.id === session.user.id && "justify-between",
+          !session || !session.user || (chefInfo.id !== session.user.id && "justify-end"),
```

Comment:
```suggestion
          session?.user && chefInfo.id === session.user.id && "justify-between",
          !session?.user || (chefInfo.id !== session.user.id && "justify-end"),
```
余談ですがオプショナルチェーンとか使えば省略できそうな気がしました

Reply:
@daiki-kon 

> 余談ですがオプショナルチェーンとか使えば省略できそうな気がしました

ありがとうございます、反映しました。
[a91cb3c](https://github.com/qin-team-recipe/08-recipe-app/pull/156/commits/a91cb3c77fa164ce42dbd9058620ad720c6727e4)